gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagActionStoreChangeMonitor.java

```diff
@@ -50,9 +51,10 @@ public class DagActionStoreChangeMonitor extends HighLevelConsumer {
   public static final String DAG_ACTION_CHANGE_MONITOR_PREFIX = "dagActionChangeStore";
 
   // Metrics
-  ContextAwareMeter killsInvoked;
-  ContextAwareMeter resumesInvoked;
-  ContextAwareMeter unexpectedErrors;
+  private ContextAwareMeter killsInvoked;
+  private ContextAwareMeter resumesInvoked;
+  private ContextAwareMeter unexpectedErrors;
+  private Meter messageProcessedMeter;
```

Comment:
why do you use normal `Meter` instead of `ContextAwareMeter`? Also we should end all the names in "Meter" or none of them ie: `messagesProcessed` or `killsInvokedMeter`